multiversx_sdk/core/transactions_factories/relayed_transactions_factory_test.py

```diff
@@ -231,3 +231,58 @@ def test_compute_relayed_v2_transaction(self):
         assert relayed_transaction.options == 0
         assert relayed_transaction.gas_limit == 60414500
         assert relayed_transaction.data.decode() == "relayedTxV2@000000000000000000010000000000000000000000000000000000000002ffff@0f@676574436f6e7472616374436f6e666967@fc3ed87a51ee659f937c1a1ed11c1ae677e99629fae9cc289461f033e6514d1a8cfad1144ae9c1b70f28554d196bd6ba1604240c1c1dc19c959e96c1c3b62d0c"
+
+    def test_relayed_v3(self):
+        alice = self.wallets["alice"]
+        alice_address = Address.new_from_bech32(alice.label)
+        bob = self.wallets["bob"]
+
+        tx = Transaction(
+            sender=bob.label,
+            receiver="erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzllls8a5w6u",
+            gas_limit=1_000_000,
+            chain_id=self.config.chain_id,
+            data=b"getContractConfig",
```

Comment:
For the sake of the example, `test` or `doSomething` instead of `getContractConfig` (sounds like a readonly / query endpoint)?